src/Bridges/Constraint/bridges/vectorize.jl

```diff
@@ -49,7 +49,7 @@ function bridge_constraint(
     if !iszero(scalar_const)
         throw(MOI.ScalarFunctionConstantNotZero{T,G,typeof(set)}(scalar_const))
     end
-    vector_f = convert(F, scalar_f)
+    vector_f = convert(F, MOI.Utilities.operate(vcat, T, scalar_f))
```

Comment:
So many assumptions. Here's a fun one: we assumed that you could call 
`convert(::AbstractVectorFunction, ::AbstractScalarFunction)`.